test/CMakeLists.txt

```diff
@@ -45,7 +45,12 @@ macro(setup_test)
     file(MAKE_DIRECTORY ${CURRENT_TEST_BINARY_DIR})
     file(GLOB TEST_FILES "${CURRENT_TEST_SOURCE_DIR}/*")
     file(COPY ${TEST_FILES} DESTINATION "${CURRENT_TEST_BINARY_DIR}/")
-    set(RUNTIME_OPTIONS "time.max_step=10 amr.plot_file=plt time.plot_interval=10 amrex.throw_exception=1 amrex.signal_handling=0")
+    set(RUNTIME_OPTIONS "time.max_step=10 amr.plot_file=plt time.plot_interval=10")
+    if(AMR_WIND_ENABLE_FPE_TRAP_FOR_TESTS)
+      set(RUNTIME_OPTIONS "${RUNTIME_OPTIONS} amrex.signal_handling=1 amrex.fpe_trap_invalid=1 amrex.fpe_trap_zero=1 amrex.fpe_trap_overflow=1")
```

Comment:
Why do we need `signal_handling` for FPE trap to work?

Reply:
https://github.com/AMReX-Codes/amrex/blob/874ccc6f3f8dedf842082816f6c8aa2dfceb5830/Src/Base/AMReX.cpp#L442-L470